atd-vzd/README.md

```diff
@@ -1,18 +1,113 @@
+- `units` - each row describes a unit, or vehicle, involved in a crash. Each unit relates to one crash record.
+- `people` - each row describes a person involved in a crash. Each person relates to one unit.
+- `charges` - each row descibes a legal charge filed by the responding law enforcement agency. Each charge relates to one person
+
+#### Design
+
+The core challenge that the Vision Zero database solves is to store CRIS data in a central repository where it can be reviewed and updated City of Austin staff. The database preserves the integrity of staff members' edits while simultaneously allowing crash record updates to flow into the database from CRIS.
+
+The editing environment is achieved by managing multiple copies of each of the `crashes`, `units`, and `people` tables, so that CRIS edits and Vision Zero staff edits remain isolated.
+
+For example, the `crashes` records are managed in three tables:
+
+- `crashes_cris`: stores crash records that are created and updated by TxdDOT CRIS through the [CRIS import ETL](../atd-etl/cris_import/README.md)
+- `crashes_edits`: stores crash record edits created by Visio Zero staff through the Vision Zero Editor web app
+- `crashes`: stores a unified version of each record which combines the values in `crashes_cris` plus any values in `crashes_edits`
+
+As pictured in the diagram below, the typical data flow for a crash record is as follows:
+
+1. A new record is inserted into the `crashes_cris` table through the [CRIS import ETL](../atd-etl/cris_import/README.md).
+2. On insert into `crashes_cris`, an "empty" copy of the record is inserted into the `crashes_edits` table. The record is inserted into the `crashes_edits` table with null values in every column except the `id`, which has a foreign key constaint referencing the `crashes_cris.id` column.
+3. At the same time, a complete copy of the record is inserted into the `crashes` table.
+4. A Vision Zero Editor user may update a crash records b updating rows in the the `crashes_edits` table. When an update is received, a trigger function coalesces each value in the `crashes_edits` table with the corresponding value in the `crashes_cris` table. The resulting record—which contains the original CRIS-provide values plus any edit values made through user edits—is applied as an update to corresponding record in the `crashes` table.
+5. Similarly, when an existing `crashes_cris` record is updated through the CRIS import ETL, the updated record is coalseced against the corresponding row in the `crashes_edits` table, and result is saved in the `crashes` table.
+6. Finally, once a record is updated in the `crashes` table, additional trigger functions apply various business rules and enrich the row with spatial attributes based on it's location. These trigger functions are reserved for values that require heavy computation—additional business rules can be applied through table views.
```

Comment:
i am being so nit picky, but "its" instead of "it's"

And I thought computation-additional was a word and not that the dash was serving the role of an em-dash or semicolon